common/src/main/java/com/microsoft/identity/common/internal/broker/AndroidBrokerAccount.java

```diff
@@ -123,6 +126,17 @@ public static AndroidBrokerAccount create(@NonNull final AccountManager accountM
                     BROKER_HOST_APP_PACKAGE_NAME,
                     AccountManager.VISIBILITY_VISIBLE
             );
+            accountManager.setAccountVisibility(
+                    account,
+                    MOCK_AUTH_APP_PACKAGE_NAME,
+                    AccountManager.VISIBILITY_VISIBLE
+            );
+            accountManager.setAccountVisibility(
+                    account,
+                    MOCK_CP_PACKAGE_NAME,
+                    AccountManager.VISIBILITY_VISIBLE
+            );
```

Comment:
Do we not need LTW here as well?

Reply:
No. LTW won't be shipping with account manager.

In fact... this is probably not needed. let me remove it.